templates/partials/header.php

```diff
@@ -6,9 +6,40 @@
 
 /** @var string $html_title */
 /** @var string|callable $page_title */
+/** @var string $url */
+/** @var string $image_url */
+/** @var string $html_description */
 /** @var ?callable $sub_head */
 /** @var ?string[] $breadcrumbs */
 
+$html_title       = $html_title ?? esc_html__( 'Translation Events', 'gp-translation-events' );
+$url              = $url ?? Urls::events_home();
+$html_description = $html_description ?? esc_html__( 'WordPress Translation Events', 'gp-translation-events' );
+$image_url        = $image_url ?? Urls::event_default_image();
+
+add_action(
+	'gp_head',
+	function () use ( $html_title, $url, $html_description, $image_url ) {
+		echo '<meta name="twitter:card" content="summary" />' . "\n";
+		echo '<meta name="twitter:site" content="@WordPress" />' . "\n";
+		echo '<meta name="twitter:title" content="' . esc_attr( $html_title ) . '" />' . "\n";
+		echo '<meta name="twitter:description" content="' . esc_attr( $html_description ) . '" />' . "\n";
+		echo '<meta name="twitter:creator" content="@WordPress" />' . "\n";
+		echo '<meta name="twitter:image" content="' . esc_url( $image_url ) . '" />' . "\n";
+		echo '<meta name="twitter:image:alt" content="' . esc_attr( $html_title ) . '" />' . "\n";
+
+		echo '<meta property="og:url" content="' . esc_url( $url ) . '" />' . "\n";
+		echo '<meta property="og:title" content="' . esc_attr( $html_title ) . '" />' . "\n";
+		echo '<meta property="og:description" content="' . esc_attr( $html_description ) . '" />' . "\n";
+		echo '<meta property="og:site_name" content="' . esc_attr( get_bloginfo() ) . '" />' . "\n";
+		echo '<meta property="og:image:url" content="' . esc_url( $image_url ) . '" />' . "\n";
+		echo '<meta property="og:image:secure_url" content="' . esc_url( $image_url ) . '" />' . "\n";
```

Comment:
I don't know if it's really necessary but I think we should show absolute URLs here.

Reply:
Done!